Cargo.toml

```diff
@@ -39,4 +38,5 @@ features = [
 ]
 
 [dev-dependencies]
-rand = "0.4"
+rand = "0.8.0"
```

Comment:
This upgrade does not compile for me. A test uses `gen_ascii_chars()` which is not available in 0.8 AFAICT.